lib/id3c/cli/command/manifest.py

```diff
@@ -295,9 +348,15 @@ def _parse(*,
         else:
             parsed_manifest[dst] = select_column(manifest, src["name"])
 
-    # Drop rows with null sample values, which may be introduced by space
-    # stripping.
-    parsed_manifest = parsed_manifest.dropna(subset = ["sample"])
+    # Drop rows that have no data for the sample_column and/or the collection_column, depending
+    # on which columns are configured. If both sample_column and collection_column are configured,
+    # drop rows if both columns don't have data.
+    if sample_column and collection_column:
+        parsed_manifest = parsed_manifest.dropna(subset = ["sample", "collection"], how='all')
+    elif sample_column:
+        parsed_manifest = parsed_manifest.dropna(subset = ["sample"])
+    else:
```

Comment:
having neither `sample_column` nor `collection_column` would fall into this edge case. I think it's better to make this `elif collection_column:` here to guard against future changes we may make.

Reply:
Good point. I've made that change.

Reply:
Thanks! Do you think we should throw an error in an `else:` statement that follows those conditionals? (I'm not sure)

Reply:
I don't think we need to throw an error here if neither sample_column nor collection_column is populated because the parse and parse_using_config functions both check that at least one of sample_column and collection_column is set.